### PR TITLE
created a shop instance method and added description migration

### PR DESCRIPTION
**Pull request by mlan080**
*Created shop instance method and insert 'Green tea cafe' entry into the database 

*Add description to table migration  by using add_column command

*Check entries successfully inserted into sqlite db 

close https://github.com/mlan080/tea-shops/issues/7

### CODE REVIEW

**Comment by actfong** [comment]
@mlan080 
What kind of assertions would you make to validate that your `create` method is behaving as it should?

**Comment by actfong** [comment]
No, the idea is not to compare name with description. 
But rather that the name you use when create should be equal to the name you retrieve out of the database

File: spec/shop_spec.rb [hunk 1]
```diff
@@ -4,14 +4,22 @@
...
   it 'responds to name' do #description of your spec/test
     shop = Shop.new
-
     expect(shop).to respond_to(:name)
     #expects shop instance to have an name accessor/method
   end
 
-  it 'sums two numbers' do
-    result = Shop.sum(3,4)
+describe "#create" do
```
Comment (actfong) [inline spec/shop_spec.rb]: indentation. the describe blocks for your methods should be nested within the describe block of your class

File: app/models/shop.rb [hunk 2]
```diff
@@ -1,7 +1,23 @@
...
+    @name = "Orange tea cafe"
+  end
+
+  def count
+    return DB[:shops].count
+  end
 
-  def self.sum(a,b) #class method
-    a+b
+  def name_matches
```
Comment (lucasrenan) [inline app/models/shop.rb]: I think this method is not supposed to exist 😅

File: app/models/shop.rb [hunk 3]
```diff
@@ -1,7 +1,23 @@
...
+  def create
+    shops = DB[:shops] # Define a dataset, table: shops
+    shops.insert(:name => name, :description => description) # Populate the table, :name is column name, name is method accessing the instance variable
+  end
+
+  def initialize
+    @name = "Orange tea cafe"
+  end
+
+  def count
```
Comment (lucasrenan) [inline app/models/shop.rb]: don't forget to add a spec for the `count` method as well

File: spec/lib/counter.rb [hunk 4]
```diff
@@ -0,0 +1,12 @@
+class Counter
```
Comment (lucasrenan) [inline spec/lib/counter.rb]: apparently, this class is not used, so you should remove it.

File: app/models/shop.rb (hunk 3 above)
Comment (lucasrenan) [reply app/models/shop.rb]: btw, `count` should be a class method.

File: spec/shop_spec.rb [hunk 5]
```diff
@@ -4,14 +4,22 @@
...
+  it 'increments data row by 1' do
+    shop = Shop.new
+    expect{shop.create}.to change{shop.count}
+  end
+end
 
-    expect(result).to eq(7)
+describe "#name_matches" do
+  it 'should be true if it matches' do
```
Comment (lucasrenan) [inline spec/shop_spec.rb]: related to the specs descriptions, `should` is not used anymore, currently, we write more like: it matches, it returns, it does, etc.

**Pull request merged**